corehq/apps/cloudcare/templates/formplayer/debugger.html

```diff
@@ -89,18 +89,34 @@
     <!--ko if: xPathQuery.errorResult()-->
     <div data-bind="text: xPathQuery.errorResult(), css: 'text-danger'"></div>
     <!--/ko-->
-    <!--ko if: options.useCodeMirror && xPathQuery.successResult()-->
+    <!--ko if: xPathQuery.isSuccessTruncated() && options.showInFull-->
+    <div>
+        <i>Showing the first </i><i data-bind="text: xPathQuery.getMaxLines()"/><i> lines...</i>
```

Comment:
Is this meant to be _italics_?  You should use the `<em>` tag instead. 
```html
<em>Showing the first <span data-bind="text: xPathQuery.getMaxLines()"/></span> lines...</em>
```

Reply:
yes, <3 semantic tags